src/chi-vue/src/components/column-customization/ColumnCustomizationModalContent.tsx

```diff
@@ -67,12 +67,12 @@ export default class ColumnCustomizationContent extends Vue {
       icon === 'chevron-up'
         ? 'Move Up'
         : icon === 'chevron-down'
-        ? 'Move Down'
-        : icon === 'chevron-left'
-        ? 'Deselect'
-        : icon === 'chevron-right'
-        ? 'select'
-        : '';
+          ? 'Move Down'
+          : icon === 'chevron-left'
+            ? 'Deselect'
+            : icon === 'chevron-right'
+              ? 'select'
+              : '';
```

Comment:
Nesting tertiary operators is not a good practice. Please, use a switch for this